swift_static_framework/swift_static_framework.bzl

```diff
@@ -90,8 +105,17 @@ def _swift_static_framework_impl(ctx):
 
     input_files = [fat_file]
     if generated_objc_hdr_file:
+        ctx.actions.write(
+            output = modulemap_file,
+            content = _modulemap_file_content(module_name)
```

Comment:
For proper formatting, I think we need a trailing comma after:

`content = _modulemap_file_content(module_name),`